app/views/dispatch/forecast.erb

```diff
@@ -0,0 +1,65 @@
+<div class="container">
+
+	<div class='row'>
+		<div class='col-xs-12'>
+			<div class='panel panel-default'>
+				<div class='panel-heading'>
+					Active Missions
+				</div>
+				<% if @active_missions.length == 0 %>
+					<div class='panel-body'>
+						<h5>No trucks out</h5>
+					</div>
+				<% else %>
+					<% @active_missions.each do |mission| %>
+						<div class='panel-body'>
+							<p class="text-success">
+								<%= "#{mission.dateTime} :: #{mission.name} #{mission.unit_serviced}" %>
+							</p>
+							<a href=<%="/mission/#{mission.id}"%>>
+								<button type="button" class="btn btn-default">
+									View
+								</button>
+							</a>	
+						</div>
+					<%end%>
+				<%end%>
+			</div>
+		</div>
+	</div>
+
+	<div class='row'>
+		<div class='col-xs-12'>
+			<div class='panel panel-default'>
+				<div class='panel-heading'>
+					Upcoming Missions
+				</div>
+				<% if @upcoming_missions.length == 0 %>
+					<div class='panel-body'>
+						<h5>Nothing Planned</h5>
+					</div>
+				<% else %>
+					<% @upcoming_missions.each do |mission| %>
+						<form action=<%="/mission/#{mission.id}"%> method='post'>
+						<% if mission.step_off < Time.now %>
+							<div class='panel-body'>
+								<p class="text-danger">
+									<%= "#{mission.dateTime} :: #{mission.name} #{mission.unit_serviced}" %>
+						<%else%>
+							<div class='panel-body' class='success'>
+								<p class='text-info'>
+								<%= "#{mission.dateTime} :: #{mission.name} #{mission.unit_serviced}" %>
```

Comment:
another reason for extracting things out is so we can re-use them (like right here! and on line 47)